docs/sdk/migrations/datil-dev.md

```diff
@@ -0,0 +1,82 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Migrating from Cayenne to Datil-dev
+
+Previously Lit networks were powered by a pre-Bedrock Optimism L2 rollup called [Chronicle](../../network/rollup.mdx). In our effort to deliver a more performant and stable backend for Lit's infrastructure, we're launching a new network, Chronicle Vesuvius, powered by [Arbitrum Orbit](https://arbitrum.io/orbit).
+
+Like `cayenne`, `datil-dev` is a centralized DevNet intended to be used for application prototyping/getting familiar with Lit. Unlike the existing chains `manzano` and `habanero`, payment is not required for usage of the Lit network.
+
+## Breaking Changes and Important Updates
+
+### New Network, New PKPs
+
+Because existing PKPs were minted on the old Chronicle chain, migration from `cayenne` to `datil-dev` requires re-minting of all PKPs. This also means transferring ownership of assets owned by PKPs minted on Chronicle, to the newly minted ones on Chronicle Vesuvius.
+
+### Encrypted Data
+
+Because Chronicle Vesuvius is a new Lit network, a new round of Distributed Key Generation (DKG) has been done, resulting in new BLS root keys for the network. This means all previously encrypted data using `cayenne`'s public BLS key, will **not** be able to be decrypted using the `datil-dev` network.
```

Comment:
It's just called Vesuvius

```suggestion
Because Vesuvius is a new Lit network, a new round of Distributed Key Generation (DKG) has been done, resulting in new BLS root keys for the network. This means all previously encrypted data using `cayenne`'s public BLS key, will **not** be able to be decrypted using the `datil-dev` network.
```